src/shared/components/annotation/genomeNexus/MutationAssessor.tsx

```diff
@@ -17,6 +17,8 @@ export function hideArrow(tooltipEl: any) {
 }
 
 export default class MutationAssessor extends React.Component<IMutationAssessorProps, {}> {
+    static MUTATION_ASSESSOR_URL:string = "http://mutationassessor.org/r3/";
```

Comment:
hi, sorry to be annoying but lets put this in URLs module so eventually it could be configured.  also, does it support https?  Pieter was just commenting on this issue.  If not, lets put a comment saying so

Reply:
@alisman good to be annoying. Ill move it over. No https support

Reply:
Is this just hyperlink?

Reply:
yes

Reply:
no need for https, so just left it there for now. It's unlikely we'll ever want to configure this in the future and it makes sense to have the URL on MutationAssessor component I think